var/spack/repos/builtin/packages/r-ctc/package.py

```diff
@@ -0,0 +1,38 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class RCtc(RPackage):
+    """Tools for export and import classification trees and clusters
+       to other programs"""
+
+    homepage = "https://www.bioconductor.org/packages/release/bioc/html/ctc.html"
+    git      = "https://git.bioconductor.org/packages/ctc.git"
+
+    version('1.54.0', commit='0c3df81dfc8fabe12e11884bed44b64e11fd6d4e')
+
+    depends_on('r-amap')
+    depends_on('r@3.5.0:3.5.9', when='@1.54.0:')
```

Comment:
Both of these should probably be `type=('build', 'run')`

Reply:
Added. Is `type=('build', 'run')` not implied by it being an `Rpackage` ?